packages/frontend-usagers/src/pages/demande-sejour/liste.vue

```diff
@@ -433,6 +445,12 @@ async function copyDS(dsId) {
     );
   }
 }
+async function deleteOrCancelDS(dsId, statut) {
```

Comment:
ici, je ne créerais pas de fonction pour ca. Souvent dans les best practices, on evite de passer des bouleens dans les pramètres (https://alexkondov.com/should-you-pass-boolean-to-functions/). Alors ok c'est pas un bouleen, mais pas loin :D

Reply:
Pris en compte, suppression de la fonction, mise en place d'une condition ternaire